Developed a Copy of the New York Times Article Page

MarilenaRoque opened the pull request:
Added all the elements and style for the Page applicating all the theories learned about the position and floating elements. Including float, grid, box model, flexbox, relative position, different display types, CSS reset.  We also made the recommended validations with Linter and W3 Validator.

#NAVBAR
Position Sticky
Included Icons, Buttons, Logo, and Links
Defined background and border for the Navbar and its elements

#Main
Embedded images and video
Position the section and its elements
Style the text
Included Media Icons
Inflation Section with the Paragraphs overlapping the image

#Asides
Position of the Elements using float and flexbox
Embedded Images 
Included and style the images caption

#BottomNavigation

Positioned all the list with its links
Included the icons
Included the borders

#Footer
Styled and positioned the footer list

MarilenaRoque committed:
First Files added

Added the index.html style.css reset.css and its initial elements

MarilenaRoque committed:
Navbar Elements and Main News Image and Icons

Added the elements for the navbar
added the title of the main new
added the image
added the head of the new with date and social media icons

Commit:
Added main elements

Adding paragraphs
Adding titles
Adding images
Adding video
Adding icons inside main

MarilenaRoque committed:
Added the sections bellow the main content

Added the Astronomy and Space section and its elements
Added the Editor's Picks section and its elements
Added the Most Popular Section and its Elements
Added the Navbar Footer and its lists and icons

Commit:
Add navbar style

Change the size of logo
Change font style
Positioning the navbar elements

Commit:
navbar and main styles

MarilenaRoque committed:
Added the font classes and position of elements

Added the font size, color and family for the firsts sections
Aligned the main picture and its caption
Aligned the first content of the article text
Aligned the items of the head-new section

Commit:
Footer styles

footer menu style
footer style

MarilenaRoque committed:
Code Format

Formated the code
Added some comments

Commit:
Inflation section format

defined a grid for the section Inflation
position all elements inside grid
apply font-style to Inflation text

MarilenaRoque committed:
Applyed style and position for Assides and Article Text

Applied style and width for the article text
Applied style for the head buttons
applied style for the assides elesments
Positioned the Assides

Commit:
Improve styles

links styles
style create account section
style figcaptions
breaklines for some text

MarilenaRoque committed:
Styled buttons and text

Added break lines for the article text
added the interaction with the buttons

MarilenaRoque committed:
Assides images responsiviness

Added responsive feature to the images within assides section

Commit:
aside style

MarilenaRoque committed:
Asside Style

Modified asside style and position

Commit:
asides modification

Commit:
change button class in css

Commit:
style icons

Commit:
Style main and aside section

new hr for aside section
Group paragraph section
Position the number in the comments icon

MarilenaRoque committed:
Position of the comments icon and README information

Added information to the Readme File
Positioned the Comment Icon
Set Up hover status for the comment icon

Commit:
linter css file

MarilenaRoque committed:
Removed Unnecessary classes

MarilenaRoque committed:
Fixed HTML Validator Requirements

Commit:
fix html and css file

Commit:
fix css reset file

rammazzoti2000 commented:
### :raised_hands: Pull Request under the review by Alexandru Bangau :raised_hands:

rammazzoti2000 commented on index.html:
```diff
@@ -0,0 +1,911 @@
+      </div>
+
+      <div class="center-nav">
+        <img alt="New York Times Logo" src="assets/images/nyt-logo.png" />
```
the image should be a link, wrap it in an anchor tag

rammazzoti2000 commented on index.html:
```diff
@@ -0,0 +1,911 @@
+
+    <div class="asides">
+         
+      <aside class="MostPopular">
```
The More Popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) should be links.

rammazzoti2000 commented on index.html:
```diff
@@ -0,0 +1,911 @@
+        </div>
+      </aside>
+
+      <aside class="SpaceAstronomy rows-asside">  
```
In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

rammazzoti2000 commented on index.html:
```diff
@@ -0,0 +1,911 @@
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">More in Space and Astronomy</a></h5>
+      <aside class="SpaceAstronomy rows-asside">
```
In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

rammazzoti2000 commented on index.html:
```diff
@@ -0,0 +1,911 @@
+      </aside>
+      <hr class="asides-hr">
+      <h5 class="bolder-titles"><a href="#">Editors’ Picks</a></h5>
+      <aside class="EditorPicks rows-asside">
```
In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))

rammazzoti2000 reviewed (changes requested):
#### Hello @MarilenaRoque and @krash46 :raised_hands: , my name is Aleaxndru and I am your code reviewer for this project, by the way awesome work :1st_place_medal: 
You did great work on this project, and have handled very well the use of flexbox and grid model.
- The readme file is descriptive, the pull request has meaningful title and description.
- The commits are short and descriptive.
- Good use of  HTML semantic :muscle: 
There are still a few issues that are preventing me from approving this project, please see the comments bellow
### :stop_sign: Required Changes :construction: :construction_worker_woman: 
- [x] The navbar logo should be a link to the [original page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0) :point_left: 
- [x] The More Popular articles on the rightmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) should be links. Wrap them in anchor tags with empty links (`href="#"`)
- [x] In the More in Astronomy and Space it should contain links to other articles with image previews on the leftmost side ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)); since the articles you have are different from the actual articles they have on the original page, use anchor tags with empty links (`href="#"`)
#### 
### <em>Kindly fix the REQUIRED issues and send another code review request.

Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>

### :stop_sign:  Status: Changes required.

rammazzoti2000 reviewed (changes requested):
### :stop_sign: Status: Required Changes

MarilenaRoque committed:
Added links for the Original Page

to the Create Login Section
to the Navbar Logo and Buttons
to all the asides articles
to the Logo and Go to Home Nav in the bottom
to the Create Login Section

MarilenaRoque committed:
Links Changes and Color Change for the Articles

Removed unnecessary links form the asides titles
Set up a color change for the Articles Itens Title

Commit:
fix style css file

Commit:
fix HTML file

added close tag for logo link
set up onclick attribute for the button

MarilenaRoque commented:
Thank you for the awesome review!

All the requirements for the approval were fitted. 

- The navbar is a link fo the original page
-The Most Popular Section is a link and also all the articles inside are links to articles on the original page.
-The Images and Subtitles in the More in Astronomy and Space are links for original articles on the original webpage

Additionally, we have added some new features, and style to the page looks more similar to the real one:

- Added functionality for the NavBar Buttons
-Added links for the Create Login Section
-Added links to the original page for the links inside the Text Container
-Set up the hover change color for the Subtitles of the Asides Articles

addod19 reviewed (approved):
## Status: Approved :trophy: :airplane: 

Hi @MarilenaRoque and @krash46, congratulations on fixing all the comments pointed out by the previous TSE :smile: On this note, I hereby approve your project :+1: However before merging to master, I have one optional comment I need you to address.
[OPTIONAL]
Kindly implement the sticky footer as you did for the nav

Good luck with your next project :+1: 
Do not forget to submit the project completion form after merging
_Happy coding_ :woman_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)
